tests/phpunit/api/v3/ContributionPageTest.php

```diff
@@ -439,10 +439,18 @@ public function testSubmitMembershipBlockIsSeparatePaymentWithEmail() {
     $this->assertTrue(in_array($membershipPayment['contribution_id'], array_keys($contributions['values'])));
     $membership = $this->callAPISuccessGetSingle('membership', array('id' => $membershipPayment['membership_id']));
     $this->assertEquals($membership['contact_id'], $contributions['values'][$membershipPayment['contribution_id']]['contact_id']);
-    $mut->checkAllMailLog(array(
-      '$ 2.00',
-      'Membership Fee',
-    ));
+    // We should have two separate email messages, each with their own amount
+    // line and no total line.
+    $mut->checkAllMailLog(
+      array(
+        'Amount: $ 2.00',
+        'Amount: $ 10.00',
+        'Membership Fee',
+      ),
+      array(
+        'Total: $'
```

Comment:
@jmcclelland still there :-(

ContributionPageTest.php:451, CommaLastItem, Priority: Normal
A comma should follow the last multiline array item. Found: 'Total: $'

No description available. Please upgrade to latest checkstyle version.